newrelic/provider.go

```diff
@@ -144,6 +144,7 @@ func Provider() terraform.ResourceProvider {
 			"newrelic_infra_alert_condition":                    resourceNewRelicInfraAlertCondition(),
 			"newrelic_insights_event":                           resourceNewRelicInsightsEvent(),
 			"newrelic_nrql_alert_condition":                     resourceNewRelicNrqlAlertCondition(),
+			"newrelic_one_dashboard":                            resourceNewRelicOneDashboard(),
```

Comment:
Did we inquire marketing and/or other sales folks on this name?  


Disclaimer: I'm on board with this resource name ✔️

Reply:
This is the name that was landed at after talking with some marketing folks.